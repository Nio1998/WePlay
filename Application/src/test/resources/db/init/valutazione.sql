
-- Tabella per le valutazioni
DROP TABLE IF EXISTS valutazione;
CREATE TABLE valutazione (
    ID INT AUTO_INCREMENT PRIMARY KEY,
    esito INT NOT NULL, -- negativa, neutra, positiva
    utente_valutato VARCHAR(50),
    utente_valutante VARCHAR(50),
    ID_evento INT
);

