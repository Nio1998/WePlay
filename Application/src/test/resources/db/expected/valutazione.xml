<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <valutazione ID='1' esito='1' utente_valutato='utente1' utente_valutante='utente2' ID_evento='1'/>
    <valutazione ID='2' esito='-1' utente_valutato='utente1' utente_valutante='utente2' ID_evento='1'/>
    <valutazione ID='3' esito='1' utente_valutato='utente1' utente_valutante='utente2' ID_evento='1'/>
</dataset>